spec/install/gemfile_spec.rb

```diff
@@ -66,4 +66,19 @@
       expect(out).to match(/You passed :lib as an option for gem 'rack', but it is invalid/)
     end
   end
+
+  context "with engine specified in symbol" do
+    it "installation succeeds" do
+      simulate_ruby_version "2.3.0" do
+        simulate_ruby_engine "jruby", "9.1.2.0" do
+          install_gemfile! <<-G
+            source "https://rubygems.org"
+            ruby "2.3.0", :engine => :jruby, engine_version: "9.1.2.0"
+          G
+
+          expect(out).to match(/Bundle complete!/)
```

Comment:
@segiddins Hi! I tried to apply your suggestion:

> the test should actually assert that installing succeeds, as it is this test could continue to pass if we just changed the wording of the error message. Changing to install_gemfile! would help, as would including a gem and verifying via expect(the_bundle).to include_gem("")

But I cannot make it pass:

```ruby
it "installation succeeds" do
  simulate_ruby_version "2.3.0" do
    simulate_ruby_engine "jruby", "9.1.2.0" do
      install_gemfile! <<-G
        source "https://rubygems.org"
        ruby "2.3.0", :engine => :jruby, engine_version: "9.1.2.0"
        gem "rack"
      G

      expect(the_bundle).to include_gems "rack"
    end
  end
end
```

```
$ rspec spec/install/gemfile_spec.rb:71
Run options:
  include {:locations=>{"./spec/install/gemfile_spec.rb"=>[71]}}
  exclude {:rubygems_master=>true, :git=>"=< 2.8.1", :rubygems=>"=< 2.6.4", :ruby=>"=< 2.3.1", :realworld=>true, :sudo=>true}

bundle install
  with engine specified in symbol
$ /Users/Juan/.rubies/ruby-2.3.1/bin/ruby -I/Users/Juan/dev/bundler/lib:/Users/Juan/dev/bundler/spec -rsupport/hax /Users/Juan/dev/bundler/exe/bundle install --retry 0 --no-color
Fetching gem metadata from https://rubygems.org/..............
Fetching version metadata from https://rubygems.org/.
Resolving dependencies...
Installing rack 2.0.1 ()
Using bundler 1.13.0.rc.2
Bundle complete! 1 Gemfile dependency, 2 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
# $? => 0

$ /Users/Juan/.rubies/ruby-2.3.1/bin/ruby -I/Users/Juan/dev/bundler/lib -e "require 'rubygems' ; require 'bundler' ; Bundler.setup()
require 'rack.rb'; puts RACK"
-e:2:in `<main>': uninitialized constant RACK (NameError)
# $? => 1
    installation succeeds (FAILED - 1)


Retried examples: 0

Failures:

  1) bundle install with engine specified in symbol installation succeeds
     Failure/Error: expect(the_bundle).to include_gems "rack"

       expected the bundle to include gems "rack" but:
           rack is not installed:
               Invoking run!("require 'rack.rb'; puts RACK", {}) failed:
               -e:2:in `<main>': uninitialized constant RACK (NameError)

       Commands:
       $ /Users/Juan/.rubies/ruby-2.3.1/bin/ruby -I/Users/Juan/dev/bundler/lib:/Users/Juan/dev/bundler/spec -rsupport/hax /Users/Juan/dev/bundler/exe/bundle install --retry 0 --no-color
       Fetching gem metadata from https://rubygems.org/..............
       Fetching version metadata from https://rubygems.org/.
       Resolving dependencies...
       Installing rack 2.0.1 ()
       Using bundler 1.13.0.rc.2
       Bundle complete! 1 Gemfile dependency, 2 gems now installed.
       Use `bundle show [gemname]` to see where a bundled gem is installed.
       # $? => 0

       $ /Users/Juan/.rubies/ruby-2.3.1/bin/ruby -I/Users/Juan/dev/bundler/lib -e "require 'rubygems' ; require 'bundler' ; Bundler.setup()
       require 'rack.rb'; puts RACK"
       -e:2:in `<main>': uninitialized constant RACK (NameError)
       # $? => 1
     # ./spec/install/gemfile_spec.rb:80:in `block (5 levels) in <top (required)>'
     # ./spec/support/helpers.rb:389:in `simulate_ruby_engine'
     # ./spec/install/gemfile_spec.rb:73:in `block (4 levels) in <top (required)>'
     # ./spec/support/helpers.rb:377:in `simulate_ruby_version'
     # ./spec/install/gemfile_spec.rb:72:in `block (3 levels) in <top (required)>'

Finished in 13.16 seconds (files took 0.25397 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/install/gemfile_spec.rb:71 # bundle install with engine specified in symbol installation succeeds
```

--

Asserting the output to have `Bundle complete!` also good to test that bundler did not raise error when parsing Gemfile, right?